src/test-utils/service-mocks.ts

```diff
@@ -204,6 +205,11 @@ export const mockTrustedClaimIssuersServiceProvider: ValueProvider<TrustedClaimI
   useValue: createMock<TrustedClaimIssuersService>(),
 };
 
+export const mockMetadataServiceProvider: ValueProvider<MetadataService> = {
```

Comment:
```suggestion
export const MockMetadataServiceProvider: ValueProvider<MetadataService> = {
```

Reply:
This is a plain JavaScript object, so camel case seems right to me

Pascal case should be used for types and classes